proposals/shared-everything-threads/Overview.md

```diff
@@ -234,33 +370,291 @@ fingerprinting. Note that:
 2. this kind of fingerprinting is likely possible without the instruction via some careful timing of
    shared access from concurrently-executing threads
 
-This issue can be discussed further here: [Should we include
-`thread.hw_concurrency`?][hw-concurrency-discussion].
+Though this functionality was considered non-essential in previous
+[discussions][hw-concurrency-discussion], those were in the context of browsers, with the
+[`hardwareConcurrency`] property available. We expect standalone engines to need some equivalent for
+general parallel programming.
```

Comment:
Hm, I remain unconvinced that this primitive is more meaningful on the level of the component model than in core Wasm. I'd rather expect the opposite.

Reply:
I agree in principle, but it's a non-starter unless we can figure out a high-performance way to make imports available to the spawned thread, including in a JS embedding. Until then, this needs to be polyfillable via Workers, so it needs to be an imported API.